modules/rubiconBidAdapter.js

```diff
@@ -557,6 +653,73 @@ function parseSizes(bid) {
   return masSizeOrdering(sizes);
 }
 
+function getDigiTrustQueryParams() {
+  function getDigiTrustId() {
+    let digiTrustUser = window.DigiTrust && (config.getConfig('digiTrustId') || window.DigiTrust.getUser({member: 'T9QSFKPDN9'}));
+    return (digiTrustUser && digiTrustUser.success && digiTrustUser.identity) || null;
+  }
+
+  let digiTrustId = getDigiTrustId();
+  // Verify there is an ID and this user has not opted out
+  if (!digiTrustId || (digiTrustId.privacy && digiTrustId.privacy.optout)) {
+    return null;
+  }
+  return {
+    id: digiTrustId.id,
+    keyv: digiTrustId.keyv,
+    pref: 0
+  };
+}
+
+function appendSiteAppDevice(request) {
+  if (!request) return;
+
+  // ORTB specifies app OR site
+  if (typeof config.getConfig('app') === 'object') {
+    request.app = config.getConfig('app');
+  } else {
+    request.site = {
+      page: utils.getTopWindowUrl()
+    }
+  }
+  if (typeof config.getConfig('device') === 'object') {
+    request.device = config.getConfig('device');
+  }
+}
+
+function addVideoParameters(data, bidRequest) {
+  if (typeof data.imp[0].video === 'object' && data.imp[0].video.skip === undefined) {
+    data.imp[0].video.skip = bidRequest.params.video.skip;
+  }
+  if (typeof data.imp[0].video === 'object' && data.imp[0].video.skipafter === undefined) {
+    data.imp[0].video.skipafter = bidRequest.params.video.skipdelay;
+  }
+  if (typeof data.imp[0].video === 'object' && data.imp[0].video.pos === undefined) {
+    data.imp[0].video.pos = bidRequest.params.position === 'atf' ? 1 : bidRequest.params.position === 'btf' ? 3 : 0;
+  }
+  if (data.imp[0].bidfloor === undefined) {
+    data.imp[0].bidfloor = parseFloat(bidRequest.params.floor) > 0.01 ? bidRequest.params.floor : 0.01;
+  }
+  if (typeof data.imp[0].video === 'object' && data.imp[0].video.w === undefined) {
```

Comment:
Can't we just use parseSizes to do this logic and grab the w / h of the video?

let size = parseSizes(bidRequest, 'video');
data.imp[0].video.w = size[0];
data.imp[0].video.h = size[1];

Reply:
implemented in latest commits